Add fallback for browsers that do not support Clipboard API

GeoffSelby opened the pull request:
This fixes #3 by adding a fallback to `execCommand()` for browsers that do not support the Clipboard API such as Safari.

This fallback creates a `ghost` textarea with the svg code to be copied as the value. This is so we can automatically select the text and copy to clipboard using `execCommand("copy")`. This `ghost` textarea is invisible and removed immediately.

GeoffSelby committed:
Add fallback for browsers that do not support Clipboard API

This fixes #3 by adding a fallback to `execCommand()` for browsers that do not support the Clipboard API such as Safari.

This fallback creates a `ghost` textarea with the svg code to be copied as the value. This is so we can automatically select the text and copy to clipboard using `execCommand("copy")`. This `ghost` textarea is invisible and removed immediately.

zaydek commented:
Thank you! This is great. I’m going to add a small test suite before I support this so I don’t break anything unintentionally. I’ll be sure to merge this just after. 👍

GeoffSelby committed:
Add fallback to ensure older versions of iOS Safari are supported

Older versions of Safari on iOS had problems selecting text programatically. This small commit adds a workaround for that by using an older, yet still supported, method of selecting text when on iOS devices.

GeoffSelby commented:
I went ahead and added a workaround to ensure support for older versions of Safari on iOS. It likely wouldn't be a problem for 99% of users but I remember reading an article about iOS having issues copying text to the clipboard in previous versions. The workaround applies to any iOS device for the sake of needing to pinpoint the exact version.

zaydek commented:
This looks great! Thank you so much.
